vagrant/provisioning/roles/arkcase-prerequisites/tasks/main.yml

```diff
@@ -212,6 +212,6 @@
     insertbefore: EOF
     marker: "# {mark} ANSIBLE MANAGED BLOCK - PdfTron"
     block: |
-      export JAVA_OPTS="$JAVA_OPTS -Djava.library.path={{ root_folder }}/data/arkcase-home/.arkcase/libraries:{{ root_folder }}/app/arkcase/lib"
```

Comment:
no more need to root/app/arkcase/lib?

Reply:
yes, we have a lot of stuff here which we don't need. We have everything in the libraries folder.

Reply:
Even if we require something its better to keep it under the libraries folder instead of having it under root/app/arkcase/lib.